src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -16,11 +20,11 @@ namespace Uno.UI.Toolkit
 #endif
 	public static class UIElementExtensions
 	{
-		#region Elevation
+#region Elevation
 
-		public static void SetElevation(this UIElement element, double Elevation)
+		public static void SetElevation(this UIElement element, double elevation)
 		{
-			element.SetValue(ElevationProperty, Elevation);
+			element.SetValue(ElevationProperty, elevation);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'element' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5093276)